app/main/views/suppliers.py

```diff
@@ -139,13 +139,24 @@ def edit_registered_address():
                 )
 
             except APIError as e:
-                error = e.message
+                abort(e.status_code)
 
-            else:
-                return redirect(url_for(".supplier_details"))
+            return redirect(url_for(".supplier_details"))
 
         http_status = 400
 
+        address_form_errors = [{
```

Comment:
Seems like this would be a nice little helper given how many wtforms we have here now. This repo is fine for now although it also seems like a good candidate for utils.

Reply:
Yep, makes sense. Have moved out to a helper.
I've not implemented it in the other views however. Mostly because they're still being worked on by other people / the helper didn't actually add much / added more lines of code by having to add `form_errors` as a variable to the templates.
Let me know what you think; I was on the fence about doing it.